internal/pkg/cli/svc_init.go

```diff
@@ -263,6 +264,19 @@ func (o *initSvcOpts) Validate() error {
 			return err
 		}
 	}
+	if len(o.sourcePaths) != 0 {
+		if o.wkldType != manifestinfo.StaticSiteType {
+			return fmt.Errorf("'--%s' must be specified with '--%s %s'", sourcesFlag, typeFlag, manifestinfo.StaticSiteType)
```

Comment:
```suggestion
			return fmt.Errorf("'--%s' must be specified with '--%s %q'", sourcesFlag, typeFlag, manifestinfo.StaticSiteType)
```
to match the help suggestion